# The United States Swine Pathogen Database: integrating veterinary diagnostic laboratory sequence data to monitor emerging pathogens of swine

Graphics and analysis files for the US-SPD manuscript:

Anderson, T.K., Inderski, B., Diel, D.G., Hause, B.M., Porter, E., Clement, T., Nelson, E.A., Bai, J., Christopher-Hennings, J., Gauger, P.C., Zhang, J., Harmon, K.M., Main, R., Lager, K.M., Faaberg, K.S. The United States Swine Pathogen Database: integrating veterinary diagnostic laboratory sequence data to monitor emerging pathogens of swine. [*bioRxiv preprint* 2021.04.16.439882](https://www.biorxiv.org/content/10.1101/2021.04.16.439882v1).

**ABSTRACT**
Veterinary diagnostic laboratories derive thousands of nucleotide sequences from clinical samples of swine pathogens such as porcine reproductive and respiratory syndrome virus (PRRSV), Senecavirus A, and swine enteric coronaviruses. In addition, next generation sequencing has resulted in the rapid production of full-length genomes. Presently, sequence data are released to diagnostic clients, but are not publicly available as data may be associated with sensitive information. However, these data can be used for field-relevant vaccines; determining where and when pathogens are spreading; and are a component in pandemic preparedness efforts. We have developed a centralized sequence database that integrates private clinical data, using PRRSV data as an exemplar, alongside publicly available genomic information. We implemented the Tripal toolkit, a collection of Drupal modules that are used to manage, visualize, and disseminate biological data stored within the Chado database schema. New sequences sourced from diagnostic labs contain at a minimum: genomic information; date of collection; collection location; and a unique identifier. Users can download annotated genomic sequences using a customized search interface that incorporates data mined from published literature; search for similar sequences using BLAST-based tools; and explore annotated reference genomes. Additionally, because the bulk of data are PRRSV sequences, custom annotation pipelines have determined genotype (Type 1 or 2), the location of open reading frames and nonstructural proteins, the occurrence of putative frame shifts, and restriction fragment length polymorphism (RFLP) classification of GP5 genes. An additional seven major swine pathogens have been curated and annotated. The database provides researchers timely access to sequences discovered by veterinary diagnosticians, allowing for epidemiological and comparative virology studies. The result will be a better understanding on the emergence of novel swine viruses, and how these novel strains are disseminated in the United States and abroad.


_Database URL:_ [https://swinepathogendb.org](https://swinepathogendb.org)

_Keywords_: RNA virus, genomic epidemiology, swine, vaccines
